tests/functional/testHeartbeat.js

```diff
@@ -105,13 +105,15 @@ describe('Backend failure', function() {
               queries.screenHome.CONTROLLER_SUBTITLE, 'text'),
           'Masking connection and location');
 
-      await backendFailureAndRestore();
+      await vpn.forceHeartbeatFailure();
 
       await vpn.waitForQuery(queries.screenHome.CONTROLLER_TITLE.visible());
-      assert.equal(
-          await vpn.getQueryProperty(
-              queries.screenHome.CONTROLLER_TITLE.visible(), 'text'),
-          'VPN is off');
+
+      await vpn.waitForCondition(async () => {
```

Comment:
Should a comment be added here that the VPN should not be disconnected because of heart beat failure?